### PR TITLE
Make commands example in filters.md to use arguments style

### PR DESCRIPTION
**Pull request by wingyplus**

### CODE REVIEW

File: docs/core-library/filters.md [hunk 1]
```diff
@@ -22,19 +22,19 @@ Unix Command Line Examples
 
 * find the index for coordinates at resolution 5
 
-     `echo 40.689167 -74.044444 | geoToH3 5`
+     `geoToH3 --resolution 1 --latitude 40.689167 --longitude -74.044444`
```
Comment (dfellis) [inline docs/core-library/filters.md]: Should be `--resolution 5`

**Comment by dfellis** [comment]
Also not sure if you're going to update the `kRing` and `hexRange` calls, too, but they are also incorrect. Thank you for doing this @wingyplus :)

**Comment by wingyplus** [comment]
Update and rebase master branch.

@dfellis  I already `kRing` and `hexRange` on my previous PR https://github.com/uber/h3/pull/315.

File: docs/core-library/filters.md (hunk 1 above)
Comment (wingyplus) [reply docs/core-library/filters.md]: @dfellis fixed

**Pull request merged**